### PR TITLE
Fix polkadot companion label CI check

### PR DESCRIPTION
**Pull request by bkchr**

### CODE REVIEW

File: .github/workflows/polkadot-companion-labels.yml [hunk 1]
```diff
@@ -14,7 +14,7 @@ jobs:
         with:
           authToken: ${{ secrets.GITHUB_TOKEN }}
           ref: ${{ github.event.pull_request.head.sha }}
-          contexts: 'continuous-integration/gitlab-check-polkadot-companion-build'
+          contexts: 'continuous-integration/check-dependent-polkadot'
```
Comment (TriplEight) [inline .github/workflows/polkadot-companion-labels.yml:17]: @s3krit is it allowed to put a list of jobs here? Substrate [depends](https://github.com/paritytech/substrate/blob/master/.gitlab-ci.yml#L582-L587) both on polkadot and cumulus companions

Comment (bkchr) [inline .github/workflows/polkadot-companion-labels.yml:17]:
```suggestion
          contexts: 'continuous-integration/gitlab-check-dependent-polkadot'
```

Comment (s3krit) [reply .github/workflows/polkadot-companion-labels.yml]: As per the docs in the [readme](https://github.com/s3krit/await-status-action) yeah, you can provide a semicolon delimited list of contexts (i.e., check names). Never tried it in practice though.

**Review by s3krit — APPROVED** [review]

File: .github/workflows/polkadot-companion-labels.yml (hunk 1 above)
Comment (bkchr) [reply .github/workflows/polkadot-companion-labels.yml]: There is currently no label for `needscumuluspr` and I'm not that convinced to add this.

**Comment by bkchr** [comment]
bot merge

**Pull request merged**